lib/frontendUtils/frontendTools.ts

```diff
@@ -5,7 +5,15 @@ export const getExampleLink = (example: Partial<IExampleLink>, full: boolean = t
     const id = idToUse ? (idToUse[0] === "#" ? idToUse : `#${idToUse}`) : "";
     // webgpu parameter. Stay safe and validate
     const params = (example.engine && example.engine==='webgpu') ? `?${example.engine}` : '';
-    return (example.type === "nme" ? "https://nme.babylonjs.com/" : "https://playground.babylonjs.com/" + (full ? "full.html" : "")) + params + id;
+    switch (example.type) {
+        case "nge":
+            return `https://nge.babylonjs.com/${id}`;

```

Comment:
Wouldn t we miss param which was in the nme address before ? Same for nme

Reply:
There are no parameters passed to nme. Just a snippet id. The only parameter supported in the playground is the webgpu flag